.github/workflows/validate.yml

```diff
@@ -0,0 +1,33 @@
+name: Validate
+
+on:
+  push:
+    branches: [master]
+  pull_request:
+    branches: [master]
+
+jobs:
+  validate:
+    strategy:
+      matrix:
+        go-version: [1.14.x, 1.15.x, 1.16.x]
+    runs-on: ubuntu-18.04
+    
+    steps:
+    - name: Install Go
+      uses: actions/setup-go@v2
+      with:
+        go-version: ${{ matrix.go-version }}
+
+    - name: Checkout Branch
+      uses: actions/checkout@v2
+      with:
+        # Resolves to empty string for push events and falls back to HEAD.
+        # See: https://github.com/actions/checkout#checkout-pull-request-head-commit-instead-of-merge-commit
+        ref: ${{ github.event.pull_request.head.sha }}
+
+    - name: Validate
+      run: |
+        ./validate.sh --nofmt
```

Comment:
In contrast to Prebid Server, we don't run with `--race` flag as that's the command in our current CI/CD process.